.travis.yml

```diff
@@ -35,4 +36,8 @@ addons:
     - dejagnu
 
 env:
-  - FORK_USER=talex5 FORK_BRANCH=containers OPAMYES=true OCAML_VERSION=latest TESTS=false
+  global:
+  - FORK_USER=talex5 FORK_BRANCH=containers OPAMYES=true OCAML_VERSION=4.05.0 OPAM_INIT=true OPAM_SWITCH=$OCAML_VERSION TESTS=false
+  matrix:
+  - BAP_RUN_VERI=false
+  - BAP_RUN_VERI=true
```

Comment:
BAP_RUN_VERI=true implicitly implies that checks are not run. I suggest to use `BAP_TASK={build,test,check,verify}` that will choose a particular task. So that we can later split it into several jobs and maybe run a sequence.